xdsl/parser/core.py

```diff
@@ -74,8 +75,11 @@
     ParametrizedAttribute,
     Data,
 )
+from xdsl.ir.affine import AffineMap
 from xdsl.utils.hints import isa
 
+from . import affine_parser as affine_parser
```

Comment:
why do you need this? Does this not work?
```suggestion
from . import affine_parser
```

Reply:
Also can you make this an absolut import? We have been using these throughiut the repo, so I would stick to it.